admin/views/school/_pupil-form.ejs

```diff
@@ -88,6 +87,14 @@ if (formData._id) {
                        name="upn"
                        value="<%= formData['upn'] %>"
                 />
+                <div class="upn-text">
```

Comment:
Please revert this change - I have a different implementation that has been approved by UX.  This should be using <detail><summary> tags.